rusk-wallet/src/bin/interactive/command_menu.rs

```diff
@@ -169,6 +169,9 @@ pub(crate) async fn online(
                 .public_key(stake_idx)
                 .expect("public key to exists in interactive mode");
 
+            let has_stake =
+                wallet.check_stake_exists(stake_pk).await.unwrap_or(false);
```

Comment:
If `check_stake_exists` return an error, then it should be propagated.

Reply:
Additionally, do we really need to add the new `check_stake_exists` when we already have the `stake_info` method?

```rust
            let has_stake = wallet
                .stake_info(stake_idx)
                .await?
                .map(|s| s.amount.is_some())
                .unwrap_or_default();
```